Modules/_io/bufferedio.c

```diff
@@ -428,6 +428,23 @@ buffered_clear(buffered *self)
     return 0;
 }
 
+#ifdef HAVE_FORK
+int
+_PyIO_buffered_at_fork_reinit(PyObject *self)
+{
+    Py_INCREF(self);
```

Comment:
Why not just do the INCREF after the if statement?

Reply:
Ah, of course. Still getting a hang of reference counting. Thanks for the tips! (This one should have been obvious).